src/main/java/com/Utils/AppProperties.java

```diff
@@ -0,0 +1,40 @@
+package com.Utils;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class AppProperties {
+
+    private  final String FILENAME = "application.properties";
+    private static final AppProperties properties_file = new AppProperties();
+    private Properties properties = new Properties();
+
+    private AppProperties() {
```

Comment:
Great idea to have this kind of functionality in a separate class :) Very pleasantly surprised by the use of the singleton pattern(i hope you knew why are you doing it like this😄) 
I would only suggest separating the logic for populating the properties class in a separate method and also create the instance of the private fields in the constructor. Also maybe rename the class to something more meaningful

Reply:
Just got a reminder from a previous comments of yours when saying "put these try catch blocks in separate method :D"  and did a research for more clean approach